Tests/ttLib/tables/data/aots/base.ttx.CFF

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ttFont ttLibVersion="3.5">
+<ttFont sfntVersion="OTTO" ttLibVersion="3.5">
```

Comment:
you probably used the "split tables" option `ttx -s font.ttf` which only writes the sftnVersion in the index ttx file.